plugins/zenoh-plugin-storage-manager/src/memory_backend/mod.rs

```diff
@@ -246,6 +251,15 @@ impl Storage for MemoryStorage {
         }
         Ok(())
     }
+
+    async fn get_all_entries(&self) -> ZResult<Vec<(String, Timestamp)>> {
+        let mut result = Vec::new();
```

Comment:
It would be more efficient to create the Vec as such (after getting the read lock on map):
```rust
let mut result = Vec::with_capacity(map.len());
```